### ISSUE TITLE
[Question] about Table detection 

### ISSUE THREAD

**Issue by phamkhactu** [PaddlePaddle/PaddleOCR#9509]
Thanks for excellence repo.

I have read the tutorial for table recognition, but in tutorial only provide the table recognition. I know if want to recognize table(OCR), you must detect region of table. I can not configs to get only bounding box(top-left, bottom-right) for table.
![image](https://user-images.githubusercontent.com/42369268/226862671-473b156a-3fe7-4c73-8875-d5a6f128ed6c.png)
I am very happy for your help to get bounding box the table.
Thanks in advance.

**Comment by johntoma** [comment]
You need to use PPStructure Layout Analysis to detect tables

**Comment by phamkhactu** [comment]
> You need to use PPStructure Layout Analysis to detect tables.
> 
> https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/ppstructure/docs/quickstart_en.md
> 
> https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/ppstructure/docs/models_list_en.md

Thanks for your point out, I will try it.

**Comment by vani-mcm** [comment]
Hii.. I have a problem regarding the output of layout analysis.
Even if I cropped my roi(table area), the layout analysis output is showing it as figure, not as table. What could be the reason?
Any idea..

**Comment by tranthuhoai3786** [comment]
@vani-mcm i get the same issue, do you have any solutions?

**Comment by phamkhactu** [comment]
> Hii.. I have a problem regarding the output of layout analysis.
> Even if I cropped my roi(table area), the layout analysis output is showing it as figure, not as table. What could be the reason?
> Any idea..

@vani-mcm Yes, I have the same problem. However, I checked again, it gave me Bounding box for table not good. It means that roi not concentrate on table(IOU), almost it gave [0,0 , w,h] for bbox

**Comment by JaesangParkTR** [comment]
@vani-mcm, @tranthuhoai3786 , the layout models only detect the areas where tables are. To parse the tables you'd need to look at: https://github.com/PaddlePaddle/PaddleOCR/blob/main/ppstructure/table/README.md

**Comment by wwolfyy** [comment]
@vani-mcm, @tranthuhoai3786 , the layout models only detect the areas where tables are. To parse the tables you'd need to look at: https://github.com/PaddlePaddle/PaddleOCR/blob/main/ppstructure/table/README.md

**Comment by webvic** [comment]
Hello
Can I use paddleOCR to extract tables from photos of Russian-language paper documents that contain perspective distortions and noise?
Could you show an example of code that does this?

Thank you in advance
